test/spec/ol/style/style.test.js

```diff
@@ -108,6 +123,9 @@ describe('ol.style.Style', function () {
       expect(original.getImage().getScale()).not.to.eql(
         clone.getImage().getScale()
       );
+      expect(original.getRenderer().toString()).not.to.eql(
+        clone.getRenderer().toString()
+      );
```

Comment:
Here I think the string representation could theoretically be the same, but the reference should be different. In this specific case it is indeed different. But I would use the reference nevertheless.